docs/book/src/multiversion-tutorial/testdata/project/controllers/cronjob_controller.go

```diff
@@ -125,14 +124,27 @@ func (r *CronJobReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	}
 
 	/*
+
+		<aside class="note">
+
+		<h1>What is this index about?</h1>
+
+		<p>The reconciler fetches all jobs owned by the cronjob for the status. As our number of cronjobs increases,
+		looking these up can become quite slow as we have to filter through all of them. For a more efficient lookup,
+		these jobs will be indexed locally on the controller's name. A jobOwnerKey field is added to the
+		cached job objects. This key references the owning controller and functions as the index. Later in this
+		document we will configure the manager to actually index this field.</p>
+
+		</aside>
+
 		Once we have all the jobs we own, we'll split them into active, successful,
 		and failed jobs, keeping track of the most recent run so that we can record it
 		in status.  Remember, status should be able to be reconstituted from the state
 		of the world, so it's generally not a good idea to read from the status of the
 		root object.  Instead, you should reconstruct it every run.  That's what we'll
 		do here.
 
-		We can check if a job is "finished" and whether it succeeded or failed using status
+		We can check if a job is "finished" and whether it Complete or Failed using status
```

Comment:
nit. Leave this as it was since the docs explain how status is used to determine if the job `Complete` or `Failed`.